app/assets/stylesheets/application.scss

```diff
@@ -1,14 +1,28 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or vendor/assets/stylesheets of plugins, if any, can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the top of the
- * compiled file, but it's generally better to create a new file per style scope.
- *
- *= require foundation_and_overrides
- *= require_tree .
- *= require_self
- */
+// fonts
+@import 'fonts/railway';
+@import 'fonts/rokkitt';
+
+// variables
+@import 'variables/colors';
+
+// third Party
+@import 'font-awesome';
+
+// foundation
+@import 'foundation/settings_overrides';
+@import 'foundation/settings';
+@import 'foundation/includes';
+
+// mixins
+@import 'mixins/bem';
+
+// blocks
+@import 'blocks/add_to_set';
+@import 'blocks/category_tabs';
+@import 'blocks/filter_container';
+@import 'blocks/flash';
+@import 'blocks/footer';
+@import 'blocks/gallery';
+@import 'blocks/home';
+@import 'blocks/records';
+@import 'blocks/top-bar';
```

Comment:
Not all of these seem to be blocks? some have a few different things in them and some are more Foundation modifications than blocks. But I think it makes sense to arrange them in here, ready for future refactoring 👍

Reply:
Yes! Something to sort out in the future :)